db/import.go

```diff
@@ -388,15 +383,19 @@ func (db *DatabaseCollectionWithUser) migrateMetadata(ctx context.Context, docid
 	}
 
 	// Persist the document in xattr format
-	value, xattrValue, marshalErr := doc.MarshalWithXattr()
+	value, syncXattr, vvXattr, marshalErr := doc.MarshalWithXattr()
 	if marshalErr != nil {
 		return nil, false, marshalErr
 	}
 
 	// Use WriteWithXattr to handle both normal migration and tombstone migration (xattr creation, body delete)
```

Comment:
```suggestion
```

This comment is out of date because we use different functions for tombstone vs non tombstone.